### DISCUSSION TITLE
Tag invalid classes in Prettier plugin

### DISCUSSION THREAD

**Discussion by zslabs** [tailwindlabs/tailwindcss#7215] [Ideas]
Thanks for releasing this! 🎉

Following up on a discussion I had opened in the main Tailwind repo https://github.com/tailwindlabs/tailwindcss/discussions/6401 would this plugin be a more appropriate spot to see if we could add an option to not only move unknown classes to the start, but introduce a way to flag them as invalid (with an eventual way to whitelist certain patterns)? 

As a user, I'd like to know when I'm using a class that's invalid or "close enough" to an actual one without having to inspect the page and see a specific style is it working as expected. 

Thanks!

**Comment by reinink** [comment]
Hey! I am moving this to a discussion since it's more of an idea than an issue.

Also, I'm not sure that showing invalid classes makes sense with the Prettier plugin for Tailwind CSS, as Prettier is not a linter. That sounds like something that would be better implement with ESLint.

**Reply by zslabs** [reply]
Appreciate you moving this over! Yep, that's what I figured as well  - but regardless it's great to see the Tailwind ecosystem expanding out to more toolsets.

Exposing that invalid list in the CLI sounds like it might make more sense.

**Comment by theMosaad** [comment]
i currently use the `tailwindcss/no-custom-classname` eslint rule from https://github.com/francoismassart/eslint-plugin-tailwindcss to achieve this.
